concepts/80 Troubleshooting/50 A Drop-Down Editor Does Not Show Data/20 AspNet.Data CreateStore Processing.md

```diff
@@ -0,0 +1,12 @@
+If you call the [DevExtreme.AspNet.Data](https://github.com/DevExpress/DevExtreme.AspNet.Data#devextreme-aspnet-data).[createStore()](https://github.com/DevExpress/DevExtreme.AspNet.Data/blob/master/docs/client-side-with-jquery.md#api-reference) method to bind a standalone editor or DataGrid/TreeList lookup column, the editor expects all data operations (including value selection) to be performed on the server. Use the [DevExtreme.AspNet.Data](https://github.com/DevExpress/DevExtreme.AspNet.Data#devextreme-aspnet-data) library to obtain correct data in the [Controller](https://docs.devexpress.com/AspNetCore/401020/devextreme-based-controls/concepts/bind-controls-to-data/api-controllers) for .NET-based applications. If you have another back-end type, adjust your server logic to return data in the proper format according to the [loadOptions](/Documentation/ApiReference/Data_Layer/CustomStore/LoadOptions/) parameters. Refer to the following article for more information: [Server-Side Data Processing](Documentation/Guide/Data_Binding/Specify_a_Data_Source/Custom_Data_Sources/#Load_Data/Server-Side_Data_Processing).
```

Comment:
"have" - is a vague/weak verb, so I changed to "work with"

"proper format according to loadOptions" - seems repetitive. If you just say "according to loadOptions", you would automatically imply some sort of a proper format. 

```suggestion
If you call the [DevExtreme.AspNet.Data](https://github.com/DevExpress/DevExtreme.AspNet.Data#devextreme-aspnet-data).[createStore()](https://github.com/DevExpress/DevExtreme.AspNet.Data/blob/master/docs/client-side-with-jquery.md#api-reference) method to bind a standalone editor or DataGrid/TreeList lookup column, the editor expects all data operations (including value selection) to be performed on the server. Use the [DevExtreme.AspNet.Data](https://github.com/DevExpress/DevExtreme.AspNet.Data#devextreme-aspnet-data) library to obtain correct data in the [Controller](https://docs.devexpress.com/AspNetCore/401020/devextreme-based-controls/concepts/bind-controls-to-data/api-controllers) for .NET-based applications. If you work with another back-end type, adjust your server logic to return data according to the [loadOptions](/Documentation/ApiReference/Data_Layer/CustomStore/LoadOptions/) parameters. Refer to the following article for more information: [Server-Side Data Processing](Documentation/Guide/Data_Binding/Specify_a_Data_Source/Custom_Data_Sources/#Load_Data/Server-Side_Data_Processing).
```